tests/unit/EntityInfo.spec.ts

```diff
@@ -136,51 +136,42 @@ describe('EntityInfo - data', () => {
 })
 
 describe('EntityInfo - company info button and tooltip', () => {
-  const params = [
+  const variations = [
     {
       businessId: 'CP1234567',
       entityType: 'CP',
-      entityStatus: 'GOODSTANDING',
+      goodStanding: true,
       buttonExists: false,
       tooltipExists: false
     },
     {
       businessId: 'BC1234567',
       entityType: 'BEN',
-      entityStatus: 'GOODSTANDING',
+      goodStanding: true,
       buttonExists: true,
       buttonDisabled: undefined,
       tooltipExists: false
     },
     {
       businessId: 'BC1234567',
       entityType: 'BC',
-      entityStatus: 'GOODSTANDING',
+      goodStanding: true,
       buttonExists: true,
       buttonDisabled: undefined,
       tooltipExists: false
     },
     {
       businessId: 'BC1234567',
       entityType: 'ULC',
-      entityStatus: 'GOODSTANDING',
+      goodStanding: true,
       buttonExists: true,
       buttonDisabled: undefined,
       tooltipExists: false
     },
     {
       businessId: 'BC1234567',
       entityType: 'BEN',
-      entityStatus: 'PENDINGDISSOLUTION',
```

Comment:
Pending Dissolution is future work. (I don't know what the flag or state will look like.)